packages/assets-controllers/src/token-prices-service/codefi-v2.ts

```diff
@@ -257,6 +249,29 @@ const DEFAULT_TOKEN_PRICE_MAX_CONSECUTIVE_FAILURES =
 
 const DEFAULT_DEGRADED_THRESHOLD = 5_000;
 
+type CryptoData = {
+  allTimeHigh: number;
+  allTimeLow: number;
+  circulatingSupply: number;
+  dilutedMarketCap: number;
+  high1d: number;
+  low1d: number;
+  marketCap: number;
+  marketCapPercentChange1d: number;
+  price: number;
+  priceChange1d: number;
+  pricePercentChange1d: number;
+  pricePercentChange1h: number;
+  pricePercentChange1y: number;
+  pricePercentChange7d: number;
+  pricePercentChange14d: number;
+  pricePercentChange30d: number;
+  pricePercentChange200d: number;
+  totalVolume: number;
+};
+
+type AddressCryptoDataMap = { [address: Hex]: CryptoData };
```

Comment:
Since the addresses here are _token_ addresses and not account addresses, should we be more clear? Also, do we need to include the word "Map" in the name of the type? This fact can be gleaned by looking at the definition, and we don't typically do this for other data types such as lists, sets, etc.

What are your thoughts on:

``` suggestion
type CryptoDataByTokenAddress = { [address: Hex]: CryptoData };
```

or

``` suggestion
type MarketDataByTokenAddress = { [address: Hex]: MarketData };
```